.chloggen/rename_configtls_structs.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

```suggestion
change_type: deprecation
```
We want to have a transition period for moving from the old names to the new names, we should call out which types we're deprecating and what they're renamed to.